intranet/apps/polls/models.py

```diff
@@ -94,6 +94,30 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        groups = get_user_model().objects.none()
+        if self.groups.exists():
+            q_objects = Q(name__in=[])
+            for g in self.groups.all():
+                q_objects.add(Q(name=g), Q.OR)
+            groups = DjangoGroup.objects.filter(q_objects)
+            users = set()
+            for g in groups:
+                users = users | set(g.user_set.all())
+            return len(users)
+        else:
+            return get_user_model().objects.count() - 1
+
+    def get_percentage_voted(self, voted, able):
+        percentage = voted / able
+        return "{:.1%}".format(percentage)
+
+    def get_voted_string(self):
+        users_voted = len(self.get_users_voted())
+        users_able = self.get_num_eligible_voters()
+        percent = self.get_percentage_voted(users_voted, users_able)
+        return str(users_voted) + " out of " + str(users_able) + "(" + percent + ") of eligible users voted in this poll."
```

Comment:
Especially with this many components of the string, it really makes sense to use `.format()`.